carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch

```diff
@@ -20,28 +20,37 @@
 -->
 
 <launch>
+  <param name="use_sim_time" value="True"/>
```

Comment:
Had to double check that boolean params aren't case sensitive but that appears to be the case, so this should be fine. Normally I'm used to seeing all lower case for bools.

Reply:
The boolean value has been corrected to lowercase